distributed/client.py

```diff
@@ -3384,7 +3384,7 @@ async def _profile(
         if isinstance(workers, (str, Number)):
             workers = [workers]
 
-        state = await self.scheduler.profile(
+        state, interval = await self.scheduler.profile(
```

Comment:
@jrbourbeau I opted not to return `(state, interval)` from `client.profile` since it would be a breaking API change, but perhaps it's worth doing?